src/components/button-toggle/button-toggle.component.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon';
+import {
+  StyledButtonToggle,
+  StyledButtonToggleLabel,
+  StyledButtonToggleInput,
+  StyledButtonToggleIcon,
+  StyledButtonToggleContentWrapper
+} from './button-toggle.style';
+import guid from '../../utils/helpers/guid';
+
+const ButtonToggle = (props) => {
+  const inputGuid = guid();
+
+  function icon() {
```

Comment:
What is the benefit in declaring these functions inside the body of the `ButtonToggle` assignment? I would consider moving where you compose these to either outside of the main component, or preferably into their own file.